### PR TITLE
[RFC] Don't propagate fixed variables through MPB

### PR DESCRIPTION
**Pull request by joehuchette**
Also includes a potential bugfix, where ``updatemodel!`` is not called at the end of ``buildInternalModel``, so any MIP related updates weren't present in e.g. Gurobi.

### CODE REVIEW

File: test/model.jl [hunk 1]
```diff
@@ -400,6 +400,24 @@ context("With solver $(typeof(solver))") do
...
 end; end; end
 
+facts("[model] Test fixed variables don't leak through MPB") do
+for solver in lp_solvers
+context("With solver $(typeof(solver))") do
+    mod = Model(solver=solver)
+    @defVar(mod, x[1:3])
+    @defVar(mod, y[k=1:2] == k)
+    buildInternalModel(mod)
+    @fact MathProgBase.getvartype(getInternalModel(mod)) => fill(:Cont, 5)
```
Comment (mlubin) [inline test/model.jl]: LP solvers don't need to implement ``getvartype``.

**Comment by joehuchette** [comment]
@mlubin apparently Cbc doesn't have ``getvartype`` supported, is it straightforward to add it?

**Comment by mlubin** [comment]
@joehuchette I believe so, let me look into it.

**Comment by mlubin** [comment]
New Cbc is tagged now.

**Pull request merged**